erpnext/patches/v13_0/update_advance_received_in_sales_order.py

```diff
@@ -0,0 +1,8 @@
+import frappe
+
+def execute():
+    docs = frappe.get_all("Sales Order", {
+        "advance_paid": ["!=", 0]
+    }, "name")
+    for doc in docs:
+        frappe.db.set_value("Sales Order", doc.name, "advance_received", 1, update_modified=False)
```

Comment:
@nmimsnikhil  can you add is_exist check for advance_received field to avoid 
 pymysql.err.InternalError: (1054, "Unknown column 'advance_received' in 'field list'"